docs/modules/commons/pages/expressions.adoc

```diff
@@ -10,14 +10,28 @@ NOTE: The expression parameters marked with *bold* are mandatory.
 
 === generate
 
-Generate expression allows to create various types of data using https://www.datafaker.net/documentation/getting-started/[Datafaker] library.
+Generates random data from various domain areas.
 
 [source, subs="+quotes"]
 ----
 `#{generate(*<expression>*)}`
 ----
 
-* *`$expression`* - data type from the chosen https://www.datafaker.net/documentation/providers/[data provider]. Specify its name, desired method and required parameters to get data you want
+* *`$expression`* - one of the https://www.datafaker.net/documentation/providers/[data providers],
+its dot-separated method and space separated parameters wrapped into single
+quptes (for majority of cases parameters are not needed, their number and
+possible values depend on the chosen data provider method).
+
+Below you can find a complete list of the supported expressions with results
+exmaples. Descriptions of the data providers and their parameters can be found
+https://www.datafaker.net/documentation/providers/[in DataFaker documentation]:
+click on the data provider name to find details.
+
+[%collapsible]
+.#Click here to view the complete list of the supported expressions#
```

Comment:
Maybe we can move it to the separate section?

<img width="187" alt="image" src="https://user-images.githubusercontent.com/17816729/196107596-782d19c6-95f5-4658-933a-71313a1dedc2.png">